.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Helm Chart
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    # only run this on push events, not in PRs
+    if: github.event_name == 'push' && github.repository == 'dagger/dagger'
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "1.20"
+
+      # - uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      #   with:
+      #     registry: ghcr.io
+      #     username: ${{ github.actor }}
+      #     password: ${{ secrets.RELEASE_DAGGER_CI_TOKEN }}
```

Comment:
Did we leave this here thinking that we will be publishing an image too?

I would remove it in this PR & maybe add it later, when we need it.